app/serializers/job_for_company_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class JobForCompanySerializer < ActiveModel::Serializer
+  attributes :id, :title, :description, :url, :location, :posted_date, :remote
+  has_many :technologies
```

Comment:
Thanks for the feedback.
I left these as they were when I picked up the project.
I think I may go through and do that to all the serializers, though. (grand total of 4 or 5)